dev/validate-config-template.yml

```diff
@@ -226,6 +226,14 @@
 # monitoring_prometheus_gateway:
 # monitoring_install_which:  # one of : prometheus, datadog, stackdriver
 
+##########################
+# Plugins
+##########################
+# plugin_repo: https://raw.githubusercontent.com/spinnaker-plugin-examples/examplePluginRepository/master/repositories.json
+# plugin_name: Armory.RandomWaitPlugin
+# plugin_extension: armory.randomWaitStage
+# plugin_version: 1.1.4
```

Comment:
This plugin version hasn't been released (will do that in a second) but will allow Orca to load the plugin without any additional extension-specific config (there aren't any `hal` commands for that config).

`Armory.RandomWaitPlugin@1.1.x` won't load in Spinnaker 1.19.x but will load in `master-latest-unvalidated`. I don't see any config options for only running against specific minor versions / the nightly build. Assuming that's somehow set on the build server itself but please let me know if I'm missing something.

Reply:
I think unless we cherry-pick this change back to the 1.19/1.18/1.17 branches, it will only run as part of the master build, so you should be good there.

Would we want to test that this passes by running a flow off of this branch before merging? (Or maybe you already did that?)

Reply:
I've confirmed locally that it spits out the right hal commands and that the test passes against my Spinnaker installation. 

Is there a way for me to test against the Jenkins you have running? Would be really great if that were possible without becoming a member of the build cops group... 😄